distribution/docs/src/main/resources/content/_metadataAttributes/NITF_TRE_PIAIMC-table.adoc

```diff
@@ -19,7 +19,7 @@
 |values 0-100 +
 else null
 |isr.cloud-cover
-|Integer
+|Double
```

Comment:
Does the PIAIMC attribute need to be updated too? 
https://github.com/codice/alliance/blob/436b3705cb6f0981e20e1f0ddd1075cda0ff9e43/catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/PiaimcAttribute.java#L122-L128
https://github.com/codice/alliance/blob/436b3705cb6f0981e20e1f0ddd1075cda0ff9e43/catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/PiaimcAttribute.java#L286-L294